### DISCUSSION TITLE
SendAzureServiceBusMessage Trigger?

### DISCUSSION THREAD

**Discussion by tunechr** [elsa-workflows/elsa-core#725] [General]
Hi 

Just checking the code out and noticed this class was marked as a Trigger. 
Wanted to double-check if this was the case?

`[Trigger(Category = "Azure Service Bus", DisplayName = "Send Service Bus Message", `

**Comment by sfmskywalker** [comment]
Good catch - this activity is not supposed to be a trigger, but an `ActionAttribute`.

**Comment by sfmskywalker** [comment]
Fixed with 286a9586ad92f6565aef6c2b4fb20e960dfc1cf8

Thanks for reporting!
